src/plugins/bubble/_base.scss

```diff
@@ -0,0 +1,432 @@
+	opacity: .65;
+	overflow: visible;
+	padding: 0;
+	position: absolute;
+	right: 0;
+	text-decoration: none;
+	top: 0;
+	width: 40px; }
+
+.wb-bubble-container .reset {
+	right: 42px; }
+
+.wb-bubble-container .minimize:focus, .wb-bubble-container .reset:focus {
+	opacity: 1;
+	outline: 1px dotted #fff;
+	outline-offset: -2px; }
+
+.wb-bubble-container .conversation {
+	margin-bottom: 15px;
+	max-height: 45vh;
+	min-height: 200px;
+	overflow-x: hidden;
+	overflow-y: auto; }
+
+.wb-bubble-container .history {
+	padding-top: 15px; }
+
+.wb-bubble-container .history::before {
+	background: -webkit-gradient(linear, left top, left bottom, color-stop(20%, #fff), to(rgba(255, 255, 255, 0)));
+	background: linear-gradient(to bottom, #fff 20%, rgba(255, 255, 255, 0) 100%);
+	content: "";
+	height: 40px;
+	left: 0;
+	pointer-events: none;
+	position: absolute;
+	top: 0;
+	width: 100%;
+	z-index: 1054; }
+
+.wb-bubble-container .controls {
+	height: 75px; }
+
+.wb-bubble-container .inputs-zone fieldset:first-child {
+	border-top: 1px solid #e5e5e5; }
+
+.wb-bubble-container .inputs-zone ul:last-child {
+	margin-bottom: 0; }
+
+.wb-bubble-container .choices input[type=radio]:checked + span {
+	color: #333; }
+
+.wb-bubble-container h4,
+.wb-bubble-container legend,
+.wb-bubble-container h4 .question a {
+	font-size: 1em;
+	line-height: 1.4375; }
+
+.wb-bubble-container .question,
+.wb-bubble-container .message,
+.wb-bubble-container label,
+.wb-bubble-container .question {
+	border-radius: 15px;
+	color: #5a5a5a;
+	font-weight: normal;
+	padding: 8px 12px;
+	width: auto; }
+
+.wb-bubble-container .question {
+	background-color: #efefef;
+	min-width: 60px;
+	position: relative; }
+
+.wb-bubble-container .message:focus {
+	-webkit-box-shadow: 0 0 4px #666;
+			box-shadow: 0 0 4px #666; }
+
+.wb-bubble-container .message,
+.wb-bubble-container label {
+	background-color: #ddd; }
+
+.wb-bubble-container .message {
+	margin-right: 15px; }
+
+.wb-bubble-container label {
+	border: 1px solid #c1c1c1;
+	font-weight: bold;
+	padding: 6px 10px; }
+
+.wb-bubble-container .avatar,
+.wb-bubble-container .question {
+	display: table-cell;
+	vertical-align: middle; }
+
+.wb-bubble-container .avatar {
+	background-color: #fff;
+	background-image: url("../assets/default-avatar.png");
+	background-position: center;
+	background-repeat: no-repeat;
+	background-size: 25px;
+	height: 30px;
+	width: 30px; }
+
+.wb-bubble-container .basic-link {
+	min-height: inherit; }
+
+.wb-bubble-mrgn {
+	margin-top: 80px; }
+
+.wb-bubble-container legend:focus {
+	outline: 1px dotted #666; }
+
+/* Chat wizard inline (full-screen) */
+.wb-bubble-contained {
+	bottom: 0;
+	-webkit-box-shadow: none;
+			box-shadow: none;
+	margin: 30px auto;
+	position: static;
+	right: 0;
+	width: 100%; }
+
+.wb-bubble-contained .conversation {
+	max-height: 70vh; }
+
+.wb-bubble-contained .minimize {
+	display: none; }
+
+.wb-bubble-contained .reset {
+	right: 0; }
+
+/* Chat wizard Media queries */
```

Comment:
Remove or replace this comment. 

Is the CSS bellow are only applicable to the bubble?